Automattic/wp-calypso
# Anchor: Add design selection UI

#### Changes proposed in this Pull Request

* Move `anchor_podcast` query logic into a separate function so we can reuse it elsewhere for testing (eventually we'll store the data in the store rather than relying on the query arg in multiple places)
* Add array of theme objects for the Anchor.fm "themes" and only show these for the Anchor flow
* Update Design Picker to look at starter sites rather than the block preview API for theme screenshot data for Anchor.fm "themes" (Hannah, Gilbert, Riley)
* Hide categories/filters for the Anchor flow
* Use the minimal grid for the Anchor flow
* Add a new `anchorHeading` prop to the design picker to show a heading outside the categories sidebar for the Anchor flow

**Visual**
<img width="1876" alt="Screen Shot 2022-04-13 at 1 12 54 PM" src="https://user-images.githubusercontent.com/2124984/163234370-a6d18097-5010-4482-9162-91be4076672c.png">


#### Testing instructions

* Switch to this PR, navigate to `/setup/?flags=signup/anchor-fm&siteSlug=[yoursite.wordpress.com]`
* The design selection step should be second in this flow
* You should see the above changes
* With the flag disabled (I don't know how to disable the flag yet, stay tuned 🙃 ) test this PR to make sure none of the changes to the design selection step/design picker are negatively affecting other flows.

Related to #62666
